drv/fe310-rtc/build.rs

```diff
@@ -0,0 +1,15 @@
+use std::env;
+use std::fs::File;
+use std::io::Write;
+use std::path::PathBuf;
+
+fn main() -> Result<(), Box<dyn std::error::Error>> {
+    let mut file = {
+        let out = PathBuf::from(env::var_os("OUT_DIR").unwrap());
+        File::create(out.join("rtc_config.rs")).unwrap()
+    };
+
+    writeln!(file, "{}", build_util::task_peripherals_str())?;
+
+    return Ok(());
+}
```

Comment:
Do you see value in having a helper function that generates the file directly instead of having each component do this? I am thinking we can avoid a lot of boilerplate code then:

```
fn main() -> Result<(), Box<dyn std::error::Error>> {
    build_util::generate_peripheral_consts_file();
}
```

Reply:
I thought about that as well. I think we'd want it to take in the name as an input, but otherwise something like that is probably a good idea. Maybe out of scope for this PR though? Not a lot of work so either way it's fine with me.

Reply:
> I think we'd want it to take in the name as an input, but otherwise something like that is probably a good idea.

You don't really need that as well. We can standardize on the name, something like "periph_consts.rs". 

> Maybe out of scope for this PR though? Not a lot of work so either way it's fine with me.

I am fine either ways.

Reply:
I'll save it for a later PR, then.

Reply:
Ack.